action.yml

```diff
@@ -35,6 +32,10 @@ runs:
   args:
     - ${{ inputs.version }}
     - ${{ inputs.additional_gems }}
+    - ${{ inputs.file_paths }}
```

Comment:
Why are we adding these as args? they automatically become ENV variables when run, they just need to be args if referenced in the docker or entrypoint.sh files. Unless I am missing something

Reply:
Good point, I need to read the documentation again. I thought you should specify all of your supported inputs here but I could be wrong.

Reply:
from what i can tell without them then they still show available when running, but from other useages out there they were only added to args when used in the docker and entrypoint files all others were just inputs, but def double check